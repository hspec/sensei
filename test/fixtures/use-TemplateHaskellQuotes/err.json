{"version":"1.0","ghcVersion":"ghc-9.10.1","span":{"file":"test/fixtures/use-TemplateHaskellQuotes/Foo.hs","start":{"line":2,"column":7},"end":{"line":2,"column":13}},"severity":"Error","code":62558,"message":["Syntax error on [| 23 |]","In the Template Haskell quotation [| 23 |]"],"hints":["Enable any of the following extensions: TemplateHaskell, TemplateHaskellQuotes\nYou may enable these language extensions in GHCi with:\n  :set -XTemplateHaskell\n  :set -XTemplateHaskellQuotes"]}
