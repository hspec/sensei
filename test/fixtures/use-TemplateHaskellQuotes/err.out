test/fixtures/use-TemplateHaskellQuotes/Foo.hs:2:7: error: [GHC-62558]
    • Syntax error on [| 23 |]
    • In the Template Haskell quotation [| 23 |]
    Suggested fix:
      Enable any of the following extensions: TemplateHaskell, TemplateHaskellQuotes

